c2cgeoportal/scaffolds/create/mapserver/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM camptocamp/mapserver:latest
```

Comment:
Shouldn't we create a docker folder =>  c2cgeoportal/scaffolds/create/docker/mapserver/Dockerfile ?